main.cpp

```diff
@@ -30,335 +30,35 @@
          just split it up into source files and provide the appropriate #include directives.
  */
 
-
-
-
-
-
-
-
 #include <iostream>
-#include "LeakedObjectDetector.h"
+#include "Pizza.h"
+#include "Guitar.h"
+#include "Airliner.h"
+#include "GuitarStore.h"
+#include "Party.h"
+#include "Wrappers.h"
+
```

Comment:
If you put these in a different order, does your code still compile and run without errors?
- [ ] test this out and fix accordingly, since your header files are missing `#include` directives.